components/brave_wallet_ui/constants/types.ts

```diff
@@ -214,6 +215,7 @@ export interface PanelState {
   signMessageData: BraveWallet.SignMessageRequest[]
   switchChainRequest: BraveWallet.SwitchChainRequest
   hardwareWalletCode?: HardwareWalletResponseCodeType
+  suggestedToken: AccountAssetOptionType | undefined
```

Comment:
```suggestion
  suggestedToken?: AccountAssetOptionType
```

Reply:
Fixed! Also updated the UI to include `Network`, `Contract Address` and the ability to view `Contract Address` on block explorer.